etc/skydive.yml.default

```diff
@@ -231,20 +231,22 @@ auth:
   analyzer_password: password
 
 etcd:
+  # server parameters
```

Comment:
There is already a PR #648 re-working the config file.  I propose to keep only one PR for the whole config refactoring. Maybe you could comment the PR #648.

Reply:
ok, closing the PR